library/src/camera_impl.cxx

```diff
@@ -263,4 +263,4 @@ vtkCamera* camera_impl::GetVTKCamera()
 {
   return this->Internals->VTKRenderer->GetActiveCamera();
 }
-};
+}
```

Comment:
I think this is needed

Reply:
This is needed as it closes the `namespace f3d::detail
{`

on line 9. Should I fix the indentation so it does not look like it is there by mistake?

Reply:
you lost me there. there is no need for this change as far as i can see